source/onboard/migrating-to-mattermost.rst

```diff
@@ -100,78 +100,113 @@ When IT adopts management of Mattermost, often they will purchase the commercial
 Migrate from Slack
```

Comment:
I'm thinking the "Migrate from Slack" section should be at the top of the page, or this may need to just be its own page linked at the top of this page. I imagine Slack is likely where a customer is coming from.

Also, the sections in this page are quite random. There are several paragraphs of marketing-related talk about "Bespoke" before we get to the section about migrating from Slack. Also, both "Migrate Mattermost server" and "Migrate from Bitnami" are referring to migrating Mattermost servers to another Mattermost server, but this is not made clear.


![image](https://github.com/mattermost/docs/assets/6913320/6bf306ad-9919-41cc-992d-5c265192a4f8)

Reply:
I agree with all of your feedback. This page hasn't been reviewed thoroughly in some time. Moving Slack up on the page makes good sense, and I'd expect bespoke systems to be near the bottom of the page after other named tools.

Reply:
We have a dedicated page for the HipChat migration guide. I'm thinking we can put the slack migration docs in a separate page just the same. This will allow us to showcase that page easier from other places

![image](https://github.com/mattermost/docs/assets/6913320/f782b27e-5dae-48d7-a20f-7ebeedb1b638)

Reply:
Thanks, @mickmister! I've split out the Slack content to its own page in my latest commit.